orographic_precipitation/fastscape_ext.py

```diff
@@ -45,11 +45,11 @@ class OrographicPrecipitation:
     # --- input variables
     latitude = xs.variable(description="latitude",
                            attrs={"units": "degrees"})
-    precip_base = xs.variable(description="background, non-orographic precipitation rate",
+    precip_base = xs.variable(dims=('y', 'x'), description="background, non-orographic precipitation rate",
```

Comment:
We should still support the uniform case, i.e.,

```suggestion
    precip_base = xs.variable(dims=[(), ('y', 'x')], description="background, non-orographic precipitation rate",
```